### PR TITLE
feat(spans): Restrict extraction of span time metrics

### PR DESCRIPTION
**Pull request by jjbayer**
Only extract `span.duration`, `span.exclusive_time` and `span.exclusive_time_light` if it is needed for a common or an addon insights module.

We wanted to show the span.duration in the ddm UI for all spans, but this need disappeared, so we can restrict it again.

### CODE REVIEW

**Review by iker-barriocanal — APPROVED** [review]

**Review by Dav1dde — APPROVED** [review]

File: relay-event-normalization/src/normalize/span/tag_extraction.rs [hunk 1]
```diff
@@ -2430,4 +2430,20 @@ LIMIT 1
...
+        }"#;
+
+        let span = Annotated::<Span>::from_json(json)
+            .unwrap()
+            .into_value()
+            .unwrap();
+
+        let tags = extract_tags(&span, 200, None, None, false, None);
+        assert_eq!(tags[&SpanTagKey::Description], "SELECT column FROM table1 WHERE another_col = %s AND yet_another_col = something_very_longggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg*");
+    }
```
Comment (jjbayer) [inline relay-event-normalization/src/normalize/span/tag_extraction.rs:2448]: Drive-by improvement: This was an integration test, but it does not need to be.
